couchdb/templates/ingress.yaml

```diff
@@ -16,6 +16,9 @@ metadata:
       {{ $key }}: {{ $value | quote }}
     {{- end }}
 spec:
+  {{- if .Values.ingress.ingressClassName }}
+  ingressClassName: {{ .Values.ingress.ingressClassName }}
```

Comment:
Not sure what the authors prefer, but I personally am preferential towards `ingress.className` instead of `ingress.ingressClassName` 🤷‍♂️ Helm has an interesting tradeoff between "namespacing" and "using what Kubernetes does verbatim"

Example:
https://github.com/colearendt/helm/blob/56bc0a48b13005b250b387d27e94d79dc96eb48f/charts/generic/values.yaml#L63

Reply:
Hi Cole, should I rename `ingress.className` into `ingress.ingressClassName` or we can merge as is ?